crates/core/src/network/handler/mod.rs

```diff
@@ -362,7 +364,69 @@ fn get_content_aspect(
     ))
 }
 
-fn get_meta_aspects(
+/// This function converts an entry into the right "meta" EntryAspect and the according
+/// base address to which it is meta, if the entry is the source entry of a meta aspect,
+/// i.e. a CRUD or link entry.
+/// If the entry is not that it returns None.
+fn entry_to_meta_aspect(entry: Entry, header: ChainHeader) -> Option<(Address, EntryAspect)> {
```

Comment:
@lucksus when i see things like this my first reaction is that we want a `From` or `TryFrom` somewhere so that the rust compiler can help us more, but it's not worth blocking this PR at all